components/CollectionComponent.js

```diff
@@ -53,7 +45,7 @@ const Collection = ({ navigation }) => {
     };
 
     // Close Add Guitar Modal
-    const onCloseModal = (state) => {
+    const toggleModal = (state) => {
```

Comment:
The convention here is to call the param either `bool` or `isOpen` to be clear that it represents a true/false state.